src/Kernel.php

```diff
@@ -102,6 +102,18 @@ public function registerBundles(): iterable
             }
         }
 
+        $additionalBundlesPath = $this->getAdditionalBundlesPath();
+        if (is_file($additionalBundlesPath)) {
+            $additionalBundles = require $additionalBundlesPath;
+            if (is_array($additionalBundles)) {
+                foreach ($additionalBundles as $bundleClass) {
+                    if (class_exists($bundleClass)) {
+                        $contents[$bundleClass] = ['all' => true];
+                    }
+                }
```

Comment:
Since we're allowing a `bundles` file, we should use the regular bundles syntax and pass envs through that instead of always setting `all`.